### PR TITLE
Add unique container names for read flow. 

### PR DESCRIPTION
**Pull request by tosan88**
Also fixing service files and their sidekicks

See also https://github.com/Financial-Times/up-service-files/pull/689

### CODE REVIEW

File: methodeapi-endpoint.service [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-ExecStop=/usr/bin/etcdctl rmdir /vulcand/backends/methodeapi
-ExecStop=/usr/bin/etcdctl rm /vulcand/frontends/methodeapi/frontend
-ExecStop=/usr/bin/etcdctl rmdir /vulcand/frontends/methodeapi
+    etcdctl set /vulcand/frontends/methodeapi/frontend '{\"Type\": \"http\", \"BackendId\": \"methodeapi\", \"Route\": \"PathRegexp(`/.*`) && Host(`methodeapi.ft.com`)\"}';"
+
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rm /vulcand/backends/methodeapi/backend'
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rm /vulcand/backends/methodeapi/servers/srv'
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rmdir /vulcand/backends/methodeapi/servers'
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rmdir /vulcand/backends/methodeapi'
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rm /vulcand/frontends/methodeapi/frontend'
```
Comment (BerniVarga) [inline methodeapi-endpoint.service]: I think we shouldn't delete the frontends. Also, can we adjust this so that vulcan-config-builder should pick it up?

**Comment by BerniVarga** [comment]
Otherwise looks good! :)

**Comment by BerniVarga** [comment]
+1

File: methodeapi-endpoint.service (hunk 1 above)
Comment (tosan88) [inline methodeapi-endpoint.service]: I removed the lines where frontends are deleted, but vcb is a bit trickier, I wouldn't experiment with that right now :)

**Comment by kovacshuni** [comment]
+1

**Pull request merged**